solutions/swb-reference/README_SOLUTIONS.md

```diff
@@ -0,0 +1,227 @@
+# SWBv2 Solutions Deployment
+
+## Deploy Solutions Implementation CloudFormation template
+- Navigate here <TODO: Enter valid URL> to the SWBv2 Solutions Implementation page and download the Cloudformation template.
+- Log into your AWS account as an Administrator where you want to deploy this stack. This role will help deploying the application and performing post-deployment steps.
+- Create a new CloudFormation stack providing this template and the necessary input parameters.
+  - Note: Please make sure CDK [bootsrapping](https://docs.aws.amazon.com/cdk/v2/guide/bootstrapping.html) has been performed for this account and region
+- You have now deployed SWBv2 successfully in your main account.
+
+### Breaking down your stack name
+While following along steps you encounter further, please note that since you're working with a pre-formed template the region short name is a random string. For example if your stack name is `swb-dev-bb5823` then:
+- `dev` is the stage name
+- `bb5823` is the region short name
+
+## Link Hosting Accounts and Exlpore SWBv2
+It is now recommended to link a hosting account by creating a CloudFormation stack in it using the `solutions/swb-reference/src/templates/onboard-account.cfn.yaml` template. If you choose to do this step later, you will need to rerun the Post-Deployment step below.
+
+## Perform Post-Deployment
+Currently Service Workbench contains some steps that can only be performed upon the completion of the main account CloudFormation stack. Here are the final few steps to complete your main account setup:<br/>
+
+1. Using the same IAM role/user you used for deploying the CloudFormation stack, create a Cloud9 environment (in the same AWS account and region) as your CloudFormation stack deployment
+   - Note: Please use instance type `m5.large` or higher for quick execution.
+2. Run the following steps on the environment:
+   - Note: Replace `<YOUR_EMAIL>` with your email address in the first line of the code snippet. This will be the IT Admin user
+
+```shell
+export EMAIL=<YOUR_EMAIL>
+```
+
+```shell
+git clone https://github.com/aws-solutions/solution-spark-on-aws.git
```

Comment:
Can these instructions be placed in a shell script ? It will make the README cleaner